pantheon/src/test/java/tech/pegasys/pantheon/cli/CommandTestAbstract.java

```diff
@@ -134,7 +134,9 @@ public void displayOutput() {
     TEST_LOGGER.info("Standard error {}", commandErrorOutput.toString());
   }
 
-  CommandLine.Model.CommandSpec parseCommand(final String... args) {
+  protected CommandLine.Model.CommandSpec parseCommand(final InputStream in, final String... args) {
```

Comment:
do you need the InputStream param?

Reply:
I don't, but something in master does, will limit the change further